dask/delayed.py

```diff
@@ -230,7 +231,14 @@ def tokenize(*args, **kwargs):
 
 
 @curry
-def delayed(obj, name=None, pure=None, nout=None, traverse=True):
+def delayed(
+    obj: Any,
+    name: Hashable = None,
+    pure: Optional[bool] = None,
+    nout: Optional[int] = None,
+    traverse: bool = True
+):
```

Comment:
If the `Delayed` class is defined after the `delayed` function, you can use `"Delayed"` (class name as string) to avoid re-ordering the code.

`def delayed(...) -> "Delayed":`

Reply:
I did not realize that!! That is cool. I'll push one more. It passes.

Reply:
please use delayed annotations